Source/KSCrash/Recording/Tools/KSDynamicLinker.c

```diff
@@ -272,6 +273,14 @@ bool ksdl_dladdr(const uintptr_t address, Dl_info* const info)
 
             for(uint32_t iSym = 0; iSym < symtabCmd->nsyms; iSym++)
             {
+                // When building with Xcode 14, symbol which n_type field equal
+                // to N_ENSYM will put the actual address to n_value field, see
+                // <mach-o/stab.h>
+                if (symbolTable[iSym].n_type == N_BNSYM ||
```

Comment:
It seems it would not build on Xcodes prior to 14

Reply:
N_BNSYM and N_ENSYM define before [xnu-2050.22.13](https://opensource.apple.com/source/xnu/xnu-2050.22.13/EXTERNAL_HEADERS/mach-o/stab.h.auto.html) version, and compatible since macOS 10.8.3 and iOS 6.0
macOS:
![image](https://user-images.githubusercontent.com/16042670/210545249-2531dac0-3ae9-4fbc-8e0e-93094686c45e.png)
iOS:
![image](https://user-images.githubusercontent.com/16042670/210545068-9182226d-c142-4ba2-8ac0-739cba898e95.png)
See: https://www.theiphonewiki.com/wiki/Kernel

I think this will not cause compatibility issues.

Reply:
Oh sorry, I've confused `usr/include/stab.h` and `usr/include/mach-o/stab.h`. Yeah, it seems OK.

Reply:
Also it seems that they skip all debug `N_STAB` symbols in [dyld implementation](https://github.com/opensource-apple/dyld/blob/3f928f32597888c5eac6003b9199d972d49857b5/src/ImageLoaderMachOClassic.cpp#L2106).

Reply:
LGTM!